### PR TITLE
Tom exarhakis

### PR DESCRIPTION
**Pull request by Tomexar**
React Project 1 - Tom Exarhakis

### CODE REVIEW

File: Project-I-Social-Card/social-card/public/index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
```
Comment (joepound) [inline Project-I-Social-Card/social-card/public/index.html]: Great to see you picking up Font Awesome!

File: Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js [hunk 2]
```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Card.css';
+
+const CardContent = () =>{
+    return(
+        <div className = "card-content">
+            <h1>Get started with React</h1>
+            <p class = 'cardp'>React makes it painless to creat interactive UIs. Design simple views for each state in your application.</p>
+            <a href="#">reactjs.org</a>
+        </div>
```
Comment (joepounddev) [inline Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js:10]: You could also set the `href` attribute to lead to `reactjs.org` as per one of the requirements.

File: Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js [hunk 3]
```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Header.css';
+import moment from 'moment';
+
+const HeaderTitle = () =>{
+    return(
+        <div className = "header-title">
+            <h1>Lambda School</h1>
+            <h2>@Lambda School &#8729; {moment().format('MMM D')}</h2>
```
Comment (joepounddev) [inline Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js:9]: To match the exact format in the design file, you can use `.format('D MMM').toLowerCase()` instead.

**Review by joepounddev — COMMENTED** [review]
Excellent work with this first project (Social Card), Tom! You completed MVP and some of the stretch goals. You also went even further by exploring Sass (SCSS) and Font Awesome. It looks like all your components have been built out well! When you have time, do try changing up the styles to make the components match the dimensions of the images for the design file - it would be a nice exercise in organizing the appearances of your components too.

It would also be great to see more commits for this project - there's a lot of files to deal with (given the numerous components you're working with), so change tracking will become even more important. Committing more often (at least every time you finish a sub-task) will help greatly in future projects.

```
Social Card

MVP
    - [X] Create a <HeaderContainer /> container component that will hold your header components.
        • [X] Create an <ImageThumbnail /> component using the provided URL as it's img src.
        • [X] Create a <HeaderTitle /> component that displays the Lambda School header text, @LambdaSchool handle and timestamp.
        • [X] Create a <HeaderContent /> component that displays the copy provided in the headers content.
    - [X] Create a <CardContainer/ > container component that will hold your card components.
        • [X] Create a <CardBanner /> component that will display the provided image as its background
        • [X] Create a <CardContent /> component that displays the card copy provided.
        • [ ] The entire <CardContainer /> should take a user to https://www.reactjs.org when clicked.
    
Stretch
    - [X] Create a <Footer/> component that pulls in the icons and displays them properly.
    - [ ] Build out the functionality so that a user can click on the icons and have them react to events.
    - [ ] Ensure that your product is as pixel perfect as possible using any tools that you were introduced to in previous weeks.
    - [X] Look up the moment.js library and figure out how to format your time-stamp in your header to be todays date.
    - Additional:
        • [X] Explored SCSS preprocessing
        • [X] Used Font Awesome icons
    
Rating: 3
```

File: Project-I-Social-Card/social-card/public/index.html (hunk 1 above)
Comment (joepounddev) [inline Project-I-Social-Card/social-card/public/index.html:7]: Great to see you picking up Font Awesome!

**Pull request merged**